ngraph/CMakeLists.txt

```diff
@@ -202,6 +200,6 @@ add_subdirectory(frontend)
 
 add_subdirectory(test)
 
-if (NGRAPH_PYTHON_BUILD_ENABLE)
```

Comment:
Is there any chance that some old clients built openvino with "-DNGRAPH_PYTHON_BUILD_ENABLE=ON' only, without enabled ENABLE_PYTHON=OFF ? Such clients will not get nGraph python bindings anymore (can be considered as incompatibility)

Reply:
it can be, but:
1. we don't support compatibility on cmake options level 
2. why users need only ngraph python? I would imagine that they need only IE API 😄

Reply:
some time ago ngraph was a standalone component and users may use to use it only, with this NGRAPH_PYTHON_BUILD_ENABLE option.
I'm ok if this is not a problem. If it is, maybe we should add somewhere `if NGRAPH_PYTHON_BUILD_ENABLE then ENABLE_PYTHON=ON`

Reply:
> some time ago ngraph was a standalone component

ngraph backends are replaced with IE ones, so ngraph as a standalone component is not quite useful for users :)